packages/renderer/src/lib/troubleshooting/TroubleshootingGatherLogs.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+import { faFileLines, faScroll } from '@fortawesome/free-solid-svg-icons';
+import Button from '../ui/Button.svelte';
+import Fa from 'svelte-fa';
+import { generateLogFileName } from '../../../../main/src/plugin/troubleshooting';
+
+let logs: string[] = [];
+
+// Save files as a zip file (we first ask the user for the dialog, and then save the files to the filepath)
+async function saveLogsAsZip() {
+  const result = await window.saveFileDialog('Save Logs as .zip', generateLogFileName('podman-desktop', 'zip'));
```

Comment:
another approach would be to expose a method 'troubleShootingGenerateLogFileName'

then from client side we ask the 'main side' instead of using a share function. I think it's too fragile like that